src/core/qgslegendrenderer.h

```diff
@@ -98,7 +98,7 @@ class CORE_EXPORT QgsLegendRenderer
      *
      * \since QGIS 3.8
```

Comment:
```suggestion
     * \since QGIS 3.8
     * \since QGIS 3.14 the python signature has changed to ``QJSoneObject exportLegendToJson( context )``
```